src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -246,7 +262,13 @@ class BinaryCompactObject : public DomainCreator<3> {
       "axis of rotation is x=0. ObjectA is located on the left and ObjectB \n"
       "is located on the right. Please make sure that your choices of \n"
       "x-coordinate locations are such that the resulting center of mass\n"
-      "is located at zero."};
```

Comment:
[optional] Could you remove the newlines above? We now auto-format help strings.

This is turning into a very long paragraph. Maybe split it before explaining the radial layers in the outer shell?